bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tcomplex.py

```diff
@@ -0,0 +1,31 @@
+# Author: Massimiliano Galli CERN  05/2019
+# Original PyROOT code by Wim Lavrijsen, LBL
+
+################################################################################
+# Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from ROOT import pythonization
+import cppyy
+
+def rsub(self, other):
+        return -self+other
+
+@pythonization()
+def pythonize_tcomplex(klass, name):
+        # Parameters:
+        # klass: class to be pythonized
+        # name: string containing the name of the class
+        if name == 'TComplex':
+
+                # implements __radd__ as equal to __add__
+                klass.__radd__ = klass.__add__
+                # implements __rsub__ by assigning the function previously defined
+                klass.__rsub__ = rsub
```

Comment:
Thanks a lot @maxgalli !
Little detail here: can you rename `rsub` to `_rsub` to flag that it is an internal function of the `_tcomplex.py` module?